<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompanyManagementWindow</class>
 <widget class="QMainWindow" name="CompanyManagementWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des sociétés</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tw_companies">
        <column>
         <property name="text">
          <string>id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nom</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Forme jur.</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Nom de la société :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_name"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Forme juridique :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_form"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_add">
            <property name="text">
             <string>Ajouter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_modify">
            <property name="text">
             <string>Modifier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_delete">
            <property name="text">
             <string>Supprimer</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_quit">
            <property name="text">
             <string>Quitter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
